packages/snap-preact-components/src/components/Molecules/Slideout/Slideout.tsx

```diff
@@ -108,6 +105,29 @@ export function Slideout(properties: SlideoutProps): JSX.Element {
 	);
 }
 
+const ButtonContent = (props: { content: string | JSX.Element | undefined; toggleActive: () => void }): JSX.Element => {
+	const { content, toggleActive } = props;
+
+	if (content && typeof content == 'string') {
+		return (
+			<div
```

Comment:
I think we should keep the `ss__slideout__button` class here to keep backwards functionality. 
Also since it's always a string here, do we need to use `dangerouslySetInnerHTML` instead of just `{content}` in the jsx div? If someone wanted to render html from a string as the button, they could pass html via the the buttonContent prop now. (and use `dangerouslySetInnerHTML` if they had to but it would be at their component)

Reply:
Maybe I misunderstood, but I thought the task was to allow the button to be anything, and not force them into the div with the ss-slideout-button class. 

From the ticket - "This is okay if we're adding button styles, but if we want to inherit button styles from a site, it makes it a little more complicated as the trigger is on this specific element." .... "I'd like the ability to change the ss__slideout__button element (from a div to something else) and also add a class or other attributes to this button."

Yeah I fought the second question for a while, but for some reason, rendering {content} inside of a jsx div just was not working when the content was converted from a string originally. It would console log out and look like jsx but it wouldnt render anything. the `dangerouslySetInnerHTML` was the work around I found to work. Happy to take another look if you want to pair on it.